dist/api_data.py

```diff
@@ -1530,6 +1538,11 @@ def __ge__(self, other):
         configure debug specific behavior on a cursor. Generally only
         used for internal testing purposes''',
         type='category', subconfig=[
+        Config('checkpoint_read_timestamp', '', r'''
+            read the checkpoint using the specified timestamp. The supplied value
+            must not be older than the checkpoint's oldest timestamp. Ignored if
+            not reading from a checkpoint''',
+            undoc=True),
```

Comment:
I wouldn't fight to the death on this, but I (continue to) support getting rid of this configuration.

Reply:
There (continue to be) tests I didn't write that need it :-)  Also, it's useful for checking that the history store checkpoint is properly matched and working correctly.